source/ddbus/meson.build

```diff
@@ -0,0 +1,35 @@
+lib_src = [
+    'attributes.d',
+    'bus.d',
+    'c_lib.d',
+    'conv.d',
+    'exception.d',
+    'package.d',
+    'router.d',
+    'simple.d',
+    'thin.d',
+    'util.d',
+]
+
+ddbus_lib = library(
+    'ddbus',
+    lib_src,
+    install: true,
+    version: meson.project_version(),
+    d_module_versions: d_mod_version,
```

Comment:
Not used right now, should be removed.

Reply:
I felt like it's nice to include this when the upstream doesn't work with meson so they know where to add versions, but fair.